doc/howtos/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,184 @@
+Using Ubuntu as the Service OS with the ACRN hypervisor
+
+
+Based on the ACRN open source project and getting started guide. This documentation is to guide people how to boot Ubuntu system as the 
+Service OS with the ACRN hypervisor. By default, in the getting started guide documentation of the ACRN open source project, Clear Linux
+is being used as the Service OS and User OS on top of the ACRN hypervisor. 
+Here we are trying to boot Ubuntu system as the Service OS and launch Clear Linux as User OS with the ACRN hypervisor on official 
+supported NUC, playing steps as below:
+
+Step #1,  Installing a Ubuntu system on the NUC with standard installing steps. 
+For example with Ubuntu 16.04.4 LTS distribution.
+Download it here: https://www.ubuntu.com/download/desktop
+Burn Ubuntu 16.04.4 image to a USB key
+Install Ubuntu system with the USB key 
+After complete installation and rebooting, you could boot Ubuntu from grub menu.
+
+Step #2,  setup proxy with you prefered ways, command line tool or system settings tool.  If you don’t need to setup proxy, you could 
+skip this step
+
+Step #3,  installing ssh service with below commands and allow “root” login over SSH optional, 
+People could just ssh into the Service OS and use 'sudo' once logged in (which would be preferable from a security standpoint), then 
+restart SSH service. Of cause, this step is not must 
+if you know how to play, just skip this step.
+# sudo apt-get install openssh-server
+# sudo service ssh status
+In /etc/ssh/sshd_config 
+Change PermitRootLogin from prohibit-password to yes
+# sudo service ssh restart
+
+Step #4, additional packages needed to install, please check the related sections of getting started guide in the ACRN open source
+project, if you don’t have an chance to setup additional packages flowing up that guid to build source. You can reference below commands
```

Comment:
Can we add a link to that particular section in the Getting Started Guide?

Reply:
I agree we should not repeat these steps in here... as a matter of fact, the list below is already outdated (missing Python3 and kconfiglib which are required to build the hypervisor).

Reply:
i have different thoughts with you,  i prefer to have one-stop information in one page.

Reply:
While "one stop information" is a nice goal, duplicating information creates a problem to keep them in sync, and as @gvancuts noted, the information is already out of sync :(

Reply:
no,  at this moment,  we need to have some little changes because release model is not very good. when the release model trend stable.  we don't need to do any modification manually

Reply:
> no, at this moment, we need to have some little changes because release model is not very good. when the release model trend stable. we don't need to do any modification manually

I don't understand this comment @ailun258 , can you elaborate some more?